scenarios.md

```diff
@@ -194,6 +194,42 @@ A deployment requires a mydb image. The mydb image is routinely updated for secu
 1. The release management system deploys the new `mydb:1.0-202002131000` image.
 1. The production orchestrator validates it's signed with the Acme Rockets production key.
 
+
+
+### Scenario #7: Malicious and Erroneous Actions
+
+#### Scenario 7.1: A Repository Compromise Occurs
+An attacker manages to compromise a repository and gain access to the repository as an administrator.   
+**Implications of this requirement:**
+1. The potential damage/risk to users in this case must be limited
+1. There must be a secure way for users to recover to a known, secure state and verify this has occurred even in the face of an attacker that can act as a man-in-the-middle on the network.
+
+#### Scenario 7.2: A Developer Discloses Their Key and/or Credentials
+A developer accidentally discloses the key they use to certify their software is authentic.  For example, the developer may have unintentionally checked their key into a github repository.  
+
+Furthermore, the developer may have scripted the means by which software is added to a repository and this script (and related credentials / secrets) have all been disclosed.
+
+**Implications of this requirement:**
+1. There should be a means to get back to a secure state.  It should not be the case that the first person to notice this action now can irrevocably control the software software distribution mechanism used by the developer.
+1. There should be some means to know what actions have been performed using the developer's key and/or credentials and when those actions occurred.
+
+#### Scenario 7.3: A Step in the Supply Chain Is Compromised
```

Comment:
Can we bump this out to it's own scenario? Scenario 7 feels focused on the credentials have been compromised; now what? 
This is another great set of scenarios where dependent content is compromised. It could be included packages, or the compiler used to compile safe packages does something unsafe.